celery/contrib/testing/worker.py

```diff
@@ -130,6 +137,8 @@ def _start_worker_thread(app,
     state.should_terminate = 0
     t.join(10)
     state.should_terminate = None
+    # worker.hub.poller gets closed so we should reset _current_loop
+    worker.hub.reset()
```

Comment:
For some reason after worker is terminated _current_loop is still in memory, but doesn't have a poller anymore. We should either call `set_event_loop(None)` or `worker.hub.reset()`. Which one is preferred?